Storage/SQL/SQLiteHistory.swift

```diff
@@ -144,6 +145,17 @@ open class SQLiteHistory {
         self.favicons = SQLiteFavicons(db: self.db)
         self.prefs = prefs
         self.notificationCenter = notificationCenter
+
+        // We report the number of visits a user has
+        // this is helpful in determining what the size of users' history visits
+        // is like, to help guide testing the migration to the
+        // application-services implementation and testing the
+        // performance of the awesomebar.
+        self.countVisits { numVisits in
```

Comment:
not a fan of running a query purely for the purpose of the counting.. sounds like wasful CPU/IO resources. I'll quickly scan if there are other places I could get the count.

That said, this only happens once, asynchronously on initialization so it should be okay